services/web-server/src/login/strategies/mozilla-auth0.js

```diff
@@ -62,9 +62,14 @@ module.exports = class MozillaAuth0 {
 
   async getUser({ userId }) {
     const personApi = await this.getPersonApi();
-    const userProfile = await personApi.getProfileFromUserId(userId);
+    const [userProfileErr, userProfile] = await tryCatch(personApi.getProfileFromUserId(userId));
     const user = new User();
 
+    if (userProfileErr) {
+      debug(`error retrieving profile from the PersonAPI`);
```

Comment:
Let's report that error, too.  If "user not found" is treated as an error, then we could check for that condition and report any *other* errors.

Reply:
My thinking was to report everything *except* userNotFound -- we might want to see, for example, if a lot of our People API calls are 500'ing, a report in Sentry.  But if a bunch of people get fired and the client scanner doesn't find them when it looks them up, that's a "normal" failure and doesn't need to be reported.

The other issue with the arrangement as written here is that it treats a 500 from the people API as a missing user, which will cause us to disable clients.

So I think the right thing to do here is to propagate this error out of `getUser`, and then down where it says `// it is likely in the service logs` use `monitor.reportError` to put it in the service logs.